python/plugins/processing/algs/gdal/buildvrt.py

```diff
@@ -82,38 +84,45 @@ def defaultFileExtension(self):
+                                                     self.tr('Resolution'),
                                                      options=self.RESOLUTION_OPTIONS,
                                                      defaultValue=0))
         self.addParameter(QgsProcessingParameterBoolean(self.SEPARATE,
-                                                        QCoreApplication.translate("ParameterVrtDestination", 'Place each input file into a separate band'),
+                                                        self.tr('Place each input file into a separate band'),
                                                         defaultValue=True))
         self.addParameter(QgsProcessingParameterBoolean(self.PROJ_DIFFERENCE,
-                                                        QCoreApplication.translate("ParameterVrtDestination", 'Allow projection difference'),
+                                                        self.tr('Allow projection difference'),
                                                         defaultValue=False))
 
         add_alpha_param = QgsProcessingParameterBoolean(self.ADD_ALPHA,
-                                                        QCoreApplication.translate("ParameterVrtDestination", 'Add alpha mask band to VRT when source raster has none'),
+                                                        self.tr('Add alpha mask band to VRT when source raster has none'),
                                                         defaultValue=False)
         add_alpha_param.setFlags(add_alpha_param.flags() | QgsProcessingParameterDefinition.FlagAdvanced)
         self.addParameter(add_alpha_param)
 
         assign_crs = QgsProcessingParameterCrs(self.ASSIGN_CRS,
-                                               QCoreApplication.translate("ParameterVrtDestination", 'Override projection for the output file'),
+                                               self.tr('Override projection for the output file'),
                                                defaultValue=None, optional=True)
         assign_crs.setFlags(assign_crs.flags() | QgsProcessingParameterDefinition.FlagAdvanced)
         self.addParameter(assign_crs)
 
         resampling = QgsProcessingParameterEnum(self.RESAMPLING,
-                                                QCoreApplication.translate("ParameterVrtDestination", 'Resampling algorithm'),
+                                                self.tr('Resampling algorithm'),
                                                 options=self.RESAMPLING_OPTIONS,
                                                 defaultValue=0)
         resampling.setFlags(resampling.flags() | QgsProcessingParameterDefinition.FlagAdvanced)
         self.addParameter(resampling)
 
+        src_nodata_param = QgsProcessingParameterString(self.SRC_NODATA,
+                                                        self.tr('Nodata value(s) for input bands'),
```

Comment:
I think you should explicitly state "space separated" here -- my gut instinct would have been to use a comma separated list